src/components/ProAmm/ProAmmPriceRange.tsx

```diff
@@ -83,55 +64,74 @@ export default function ProAmmPriceRange({
+              //
+            }}
+            onRightRangeInput={() => {
+              //
+            }}
+            interactive={false}
+          />
+        </div>
         <RowBetween style={{ gap: '12px' }}>
-          <PriceRangeCard width="48%" padding="12px 8px">
-            <AutoColumn gap="10px" justify="center">
-              <Flex>
-                <Text fontSize="12px" fontWeight={500} color={theme.subText}>
-                  <Trans>MIN PRICE</Trans>
-                </Text>
-                <InfoHelper
-                  text={t`Your position will be 100% composed of ${baseCurrency?.symbol} at this price`}
-                  placement={'right'}
-                  size={12}
-                ></InfoHelper>
-              </Flex>
-              <Text textAlign="center" fontWeight="500" fontSize="20px">{`${formatTickPrice(
-                priceLower,
-                ticksAtLimit,
-                Bound.LOWER,
-              )}`}</Text>
-              <Text textAlign="center" fontSize="12px" fontWeight="500" color={theme.subText}>
-                <Trans>
-                  {quoteCurrency.symbol} per {baseCurrency.symbol}
-                </Trans>
-              </Text>
-            </AutoColumn>
-          </PriceRangeCard>
-          <PriceRangeCard width="48%" padding="12px 8px">
-            <AutoColumn gap="10px" justify="center">
-              <Flex>
-                <Text fontSize="12px" fontWeight={500} color={theme.subText}>
-                  <Trans>MAX PRICE</Trans>
-                </Text>
-                <InfoHelper
-                  text={t`Your position will be 100% composed of ${quoteCurrency?.symbol} at this price`}
-                  placement={'right'}
-                  size={12}
-                ></InfoHelper>
-              </Flex>
-              <Text fontSize="20px" textAlign="center" fontWeight="500">{`${formatTickPrice(
-                priceUpper,
-                ticksAtLimit,
-                Bound.UPPER,
-              )}`}</Text>
-              <Text textAlign="center" fontSize="12px" fontWeight="500" color={theme.subText}>
-                <Trans>
-                  {quoteCurrency.symbol} per {baseCurrency.symbol}
-                </Trans>
-              </Text>
-            </AutoColumn>
-          </PriceRangeCard>
+          <Flex>
+            <Text fontSize="12px" fontWeight={500} color={theme.subText}>
+              <Trans>Min Price</Trans>
+            </Text>
+            <InfoHelper
+              text={t`Your position will be 100% composed of ${baseCurrency?.symbol} at this price`}
+              placement={'right'}
+              size={12}
+            ></InfoHelper>
+          </Flex>
+
+          <Text fontWeight="500" fontSize="12px">
+            {`${formatTickPrice(priceLower, ticksAtLimit, Bound.LOWER)}`} {quoteCurrency.symbol} per{' '}
+            {baseCurrency.symbol}
+          </Text>
+        </RowBetween>
+        <RowBetween style={{ gap: '12px' }}>
```

Comment:
We use `gap` everywhere, so imo add this property to the `Flex` component.

Reply:
using Flex need to add justify-content and alignItems
RowBetween also has that attributes